ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.12.yaml

```diff
@@ -11,14 +11,18 @@ base_images:
     name: test
     namespace: ocp-kni
     tag: dev-scripts
+  openstack-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: openstack-installer
```

Comment:
I guess this image is not needed as we don't have OSP profiles in this config file.

Reply:
Import unused images cost time/storage/$$.

Reply:
@JianLi-RH would you mind dropping the image?